app/views/welcome/index.html.slim

```diff
@@ -75,4 +76,4 @@
               p = tweet.text
               footer
                 cite
-                  = mustdown t('.tweet_infos'), tweet
```

Comment:
Before:

![Screenshot_2021-11-06_22-31-34](https://user-images.githubusercontent.com/1866809/140624456-c259b268-d22c-44dd-96a5-9d28f70fabae.png)


After:

![Screenshot_2021-11-06_22-32-01](https://user-images.githubusercontent.com/1866809/140624459-ac47f602-c092-47d4-aea7-847aec1bed3e.png)

(it is the same)